.kitchen.yml

```diff
@@ -77,10 +79,23 @@ platforms:
 suites:
 - name: default
   run_list:
+  - recipe[os_prepare]
+  - recipe[audit]
   - recipe[apt]
   - recipe[yum]
-  - recipe[os_prepare]
+  verifier:
+    inspec_tests:
+      - test/integration/default
   attributes:
+    audit:
+      attributes:
+        audit_attribute: 'Attribute Override!'
+      reporter: 'json-file'
+      json_file:
+        location: /tmp/json_export.json
+      profiles:
+        - name: integration
+          url: https://github.com/jquick/profile_with_attributes/archive/master.zip
```

Comment:
Could we source a profile out of the inspec/inspec repo, instead of jquick's personal repos? :P)

Reply:
For sure 😆